nuxeo-core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/message/BulkCommand.java

```diff
@@ -90,6 +97,14 @@ public String getAction() {
         return action;
     }
 
+    public String getScroller() {
+        return scroller == null ? DEFAULT_SCROLLER : scroller;
```

Comment:
wdyt if we replace it by ` return Objects.requireNonNullElse(scroller, DEFAULT_SCROLLER)` if you agree you will have some adaptation on 10.10 (java 8)

Reply:
I'm not sure that's really more readable...

Reply:
Same I don't see the point